lib/cocoapods/installer.rb

```diff
@@ -49,6 +49,8 @@ class Installer
 
     include Config::Mixin
 
+    MASTER_SPECS_REPO_GIT_URL = 'https://github.com/CocoaPods/Specs.git'.freeze
```

Comment:
do we not have this defined anywhere else? I would think we still do but I guess it's possible we don't since we switched to CDN by default

Reply:
we dont! neither in Core!